src/constants/docsComponentMapping.js

```diff
@@ -124,3 +131,72 @@ export const components = {
   br: styled.br``,
   small: TextComponents.Small,
 };
+
+/**
+ * Template-specific overrides to these default styles
+ */
+
+const { td: TD } = components;
+
+export const documentationComponents = {
+  ...components,
+  wrapper: WrapperDocumentation,
```

Comment:
I moved wrappers into their own component files, and moved the template-specific component definitions to all live in here